.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Past suggestions have been to have something like this in your _global_ .gitignore - let me know if you disagree - unless there's a build process that excludes this file, then I don't care

Reply:
Sounds like a better idea to define this as part of the global gitignore :+1: great suggestion